src/sweeps/config/schema.json

```diff
@@ -639,6 +639,12 @@
         "target": {
           "type": "number",
           "description": "The sweep will finish once any run achieves this value"
+        },
+        "impute": {
+          "type": "string",
+          "description": "Metric value to use in bayes search for runs that fail, crash, or are killed",
+          "enum": ["latest", "best", "worst"],
+          "default": "worst"
```

Comment:
default: best?

Reply:
i think we should keep it backwards compatible for now (i.e., keep worst, since thats what we effectively have now) and deploy with default: best in our shadow sweep service (anaconda1). That way we can see how it does on a prod data stream without actually serving data to users

Reply:
meanwhile we can tell the concerned parties to use `{"impute": "best"}` for now

Reply:
sgtm